x/cronos/keeper/keeper.go

```diff
@@ -298,8 +299,17 @@ func (k Keeper) onPacketResult(
 		return err
 	}
 	gasLimit := k.GetParams(ctx).MaxCallbackGas
-	_, _, err = k.CallEVM(ctx, &contractAddr, data, big.NewInt(0), gasLimit)
-	return err
+	_, rsp, err := k.CallEVM(ctx, &contractAddr, data, big.NewInt(0), gasLimit)
+	if err != nil {
+		return err
+	}
+
+	if stateDB, ok := ctx.Value("statedb").(vm.StateDB); ok {
+		for _, l := range rsp.Logs {
+			stateDB.AddLog(l.ToEthereum())
```

Comment:
Interesting, so that mean we will be ablet to get the CRC20 logs for the indexer for ibc transaction?

Reply:
yea